indico/modules/events/abstracts/blueprint.py

```diff
@@ -44,6 +46,9 @@
 _bp.add_url_rule('/abstracts/<int:abstract_id>/abstract.pdf', 'display_abstract_pdf_export', RHDisplayAbstractExportPDF)
 _bp.add_url_rule('/abstracts/<int:abstract_id>/attachments/<file_id>/<filename>', 'download_attachment',
                  RHAbstractsDownloadAttachment)
+_bp.add_url_rule('/call-for-abstracts/reviewing', 'display_reviewable_tracks', RHDisplayReviewableTracks)
+_bp.add_url_rule('/call-for-abstracts/reviewing/track/<int:track_id>', 'display_reviewable_track_abstracts',
```

Comment:
I don't think the `/track` part is really needed unless we have anything else inside `/reviewing/` that's not a track